### ISSUE TITLE
Ktorfit development

### ISSUE THREAD

**Issue by Foso** [Foso/Ktorfit#102]
Do you have question on how to build the Ktorfit project? Feel free to ask them here.
I wrote a general overview [here](https://github.com/Foso/Ktorfit/blob/master/docs/architecture.md)

**Comment by ferenczAndras** [comment]
Hi. I wanted to ask a question regarding StringValues. I have to send an array of id's as formdata. 
This is my implementation:

```
        val groups: StringValues = StringValuesSingleImpl(
            caseInsensitiveName = false,
            name = "groupIds",
            values = groupIds
        )

        val bodyParameters = Parameters.build {
            appendAll(groups)
        }

        val result = provider.eventApi.addGroups(
            eventId = eventId,
            authToken = authHeader.value,
            body = FormDataContent(bodyParameters)
        )
```
where` eventId: String, groupIds: List<String>`

THE API works, if I send 2 or more data inside the groupIds field. But the server get's a different format if there is only 1 value in the list. Any advice?

**Comment by silveryTitan** [comment]
Hello, I would like to ask: If I want to replace a subdomain, is it appropriate to use the Path scheme? For example: www.google.com wants to replace 'www' with 'user'

**Comment by Foso** [comment]
> Hello, I would like to ask: If I want to replace a subdomain, is it appropriate to use the Path scheme? For example: [www.google.com](http://www.google.com) wants to replace 'www' with 'user'

Do you mean the Path annotation? No, i can only change the path

**Comment by silveryTitan** [comment]
@Foso yes, so if i want to replace the subdomain, how can i achieve it ?  Please advise. thanks

**Comment by Foso** [comment]
@silveryTitan You can do it like that https://github.com/Foso/Ktorfit/discussions/769
